apps/s_server.c

```diff
@@ -1543,6 +1556,11 @@ int s_server_main(int argc, char *argv[])
     }
 #endif
 
+    if (max_send_fragment > SSL3_RT_MAX_PLAIN_LENGTH) {
+        BIO_printf(bio_err, "Bad max send fragment size\n");
```

Comment:
output prog name here, too.  I know it's not always done, but let's get in the habit of fixing these.